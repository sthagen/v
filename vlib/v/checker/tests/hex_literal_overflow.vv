s := 'this hex character literal, \xffffffff, should be caught by the checker'
println(s)
