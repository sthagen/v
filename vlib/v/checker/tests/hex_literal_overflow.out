vlib/v/checker/tests/hex_literal_overflow.vv:1:7: error: hex character literal overflows string
    1 | a := '\x11ffff'
      |       ~~~~~~~~
    2 | b := '\x20ffff'
    3 | c := '\x10fffff'
vlib/v/checker/tests/hex_literal_overflow.vv:2:7: error: hex character literal overflows string
    1 | a := '\x11ffff'
    2 | b := '\x20ffff'
      |       ~~~~~~~~
    3 | c := '\x10fffff'
    4 | println(a)
vlib/v/checker/tests/hex_literal_overflow.vv:3:7: error: hex character literal overflows string
    1 | a := '\x11ffff'
    2 | b := '\x20ffff'
    3 | c := '\x10fffff'
      |       ~~~~~~~~~
    4 | println(a)
    5 | println(b)
